contracts/mesh-consumer/src/contract.rs

```diff
@@ -70,12 +81,22 @@ pub fn execute_receive_rewards(
             validator,
             total_funds: coin,
         })?,
-        timeout,
+        timeout: build_timeout(deps.as_ref(), &env)?,
     };
 
     Ok(Response::default().add_message(msg))
 }
 
+pub fn execute_update_packet_lifetime(
+    deps: DepsMut,
+    time: u64,
+) -> Result<Response, ContractError> {
+    // TODO: do permissions check
```

Comment:
If this is a `TODO` would rather merge without the ability to update the packet time, just have a sensible default. This is a vulnerability as is.

Reply:
Please remove for now and then this is LGTM from me.

Reply:
I placed it in comments, so we can just uncomment in the future if needed or delete.